clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -4005,6 +4005,13 @@ def SinOp : UnaryFPToFPBuiltinOp<"sin", "SinOp">;
 def SqrtOp : UnaryFPToFPBuiltinOp<"sqrt", "SqrtOp">;
 def TruncOp : UnaryFPToFPBuiltinOp<"trunc", "FTruncOp">;
 
+def AbsOp : CIR_Op<"abs", [Pure, SameOperandsAndResultType]> {
+  let arguments = (ins CIR_IntType:$src, I1Attr:$is_int_min_poison);
+  let results = (outs CIR_IntType:$result);
+  let summary = "libc builtin equivalent abs, labs, llabs";
+  let assemblyFormat = "$src `:` type($src) attr-dict";
```

Comment:
Seems like the assembly format does not explicitly cover `$is_int_min_poison`.

If you define `$is_int_min_poison` as a `UnitAttr`, you could define the assembly format as:

```
let assemblyFormat = [{
  $src ( `poison_int_min` $is_int_min_poison^ )? `:` type($src) attr-dict
}];
```